Rendering/Shader/D3D/OffShader.hlsl

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright (c) 2021 Technical University of Munich
+Chair of Computational Modeling and Simulation.
+
+TUM Open Infra Platform is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License Version 3
+as published by the Free Software Foundation.
+
+TUM Open Infra Platform is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+cbuffer WorldBuffer
+{
+	float4x4 viewProjection;
+	float4x4 projection;
+	float4x4 view;
+	float3 cam;
+};
+
+struct ApplicationToVertex
+{
+	float3 position : position;
+	float3 color    : COLOR;
+	float3 normal   : NORMAL;
+};
```

Comment:
We have only a position, no colors or normals!

Reply:
Resolved: 5f612a2